topics/transcriptomics/tutorials/srna/tours/tour.yaml

```diff
@@ -0,0 +1,771 @@
+      reads in addition to adapter removal</b> 0</li> <li><b>Overlap with
+      adapter sequence required to trim a sequence</b> 6</li> <li><b>Discard
+      reads that became shorter than length INT</b> 12</li> <li><b>Generate a
+      report file</b> Yes</li> </ul>
+    position: left
+  - title: Adaptor trimming
+    content: >-
+      Repeat for the <i>Symplekin</i> RNAi dataset collection.
+
+      <br><br>We don’t want to trim for quality because the adapter-trimmed
+      sequences represent a full small RNA molecule, and we want to maintain the
+      integrity of the entire molecule. We increase the minimum read length
+      required to keep a read because small RNAs can potentially be shorter than
+      20 nt (the default value). We can check out the generated report file for
+      any sample and see the command for the tool, a summary of the total reads
+      processed and number of reads with an adapter identified, and histogram
+      data of the length of adaptor trimmed. We also see that a very small
+      percentage of low-quality bases have been trimmed
+    backdrop: true
+  - title: Adaptor trimming
+    content: >-
+      Run <b>FastQC</b> on each collection of trimmed FASTQ read files to assess
+      whether adapters were successfully removed.
+    backdrop: true
+  - title: Questions
+    content: |-
+      <ul>
+        <li>What is the read length?</li>
+        <li>Are there any adaptors present in these reads? Which one(s)?</li>
+      </ul>
+    backdrop: false
+  - title: Adaptor trimming
+    content: >-
+      An interesting thing to note from our FastQC results is the Sequence
+      Length Distribution results. While many RNA-seq experiments have normal
+      distribution of read lengths, an unusual spike at 22nt is observed in our
+      data. This spike represents the large set of endogenous siRNAs that occur
+      in the cell line used in this study, and is actually confirmation that our
+      dataset captures the biological molecule we are interested in.
+    backdrop: true
+  - title: Adaptor trimming
+    content: >-
+      Now that we have converted to fastqsanger format and trimmed our reads of
+      the Illumina Universal Adaptors, we will align our trimmed reads to
+      reference Drosophila rRNA and miRNA sequences (dm3) to remove these
+      artifacts. Interestingly, Drosophila have a short 2S rRNA sequence that is
+      30nt long and typically co-migrates with sRNA populations during gel
+      electrophoresis. rRNAs make up a very large proportion of all non-coding
+      RNAs, and thus need to be removed. Oftentimes, experimental approaches can
+      be utilized to deplete or avoid capture of rRNAs, but these methods are
+      not always 100% efficient. We also want to remove any miRNA sequences, as
+      these are not relevant to our analysis. After removing rRNA and miRNA
+      reads, we will analyze the remaining reads as siRNA and piRNA sequences.
```

Comment:
If you don't want to change the datasets you could change this to
"we will analyze the remaining reads, which may be siRNA or piRNA sequences."